@here/harp-datasource-protocol/package.json

```diff
@@ -2,15 +2,15 @@
     "name": "@here/harp-datasource-protocol",
     "version": "0.13.0",
     "description": "Components used for the decoding and styling of data that is used by the Datasources.",
-    "main": "index.js",
-    "typings": "index",
+    "main": "dist/cjs/index.js",
+    "typings": "dist/cjs/index",
     "directories": {
         "test": "test"
     },
     "scripts": {
         "test": "cross-env mocha --require source-map-support/register $EXTRA_MOCHA_ARGS ./test/*.js",
```

Comment:
How about test files ?
Not sure, but lookg like they are generated by `tsconfig` if yes then probably not into `./test/..` but probably into `dist/cjs/test/` and ... possibly even land in final package.